.storybook/preview.tsx

```diff
@@ -165,9 +119,65 @@ const analyticsSettingsDecorator = (Story, context) => {
   return <Story {...context} />;
 };
 
-export const decorators = [
-  onDarkDecorator,
-  languageSettingDecorator,
-  analyticsSettingsDecorator,
-  themeSettingDecorator,
-];
+const preview: Preview = {
+  globalTypes: {
+    language: {
+      name: 'Language',
+      description: 'Internationalization language',
+      defaultValue: 'en',
+      toolbar: {
+        icon: 'globe',
+        items: [
+          { value: 'en', title: 'English' },
+          { value: 'es', title: 'Español' },
+        ],
+      },
+    },
+    analytics: {
+      name: 'Analytics',
+      description: 'Analytics settings',
+      defaultValue: 'off',
+      toolbar: {
+        icon: 'graphline',
+        items: [
+          { value: 'on', left: 'Analytics', title: 'Log to Actions' },
+          { value: 'off', left: 'Analytics', title: 'Off' },
```

Comment:
The "Analytics" prefixes are rendering weird. We could just remove them. Hopefully the menu title gives enough context.